modules/nw-multus-vlan-object.adoc

```diff
@@ -0,0 +1,81 @@
+// Module included in the following assemblies:
+//
+// * networking/multiple_networks/configuring-additional-network.adoc
+
+//37.1. VLAN overview
+//
+:_content-type: REFERENCE
+[id="nw-multus-vlan-object_{context}"]
+= Configuration for an VLAN additional network
+
+The following object describes the configuration parameters for the VLAN CNI plugin:
+
+.VLAN CNI plugin JSON configuration object
+[cols=".^2,.^2,.^6",options="header"]
+|====
+|Field|Type|Description
+
+|`cniVersion`
+|`string`
+|The CNI specification version. The `0.3.1` value is required.
+
+|`name`
+|`string`
+|The value for the `name` parameter you provided previously for the CNO configuration.
+
+|`type`
+|`string`
+|The name of the CNI plugin to configure: `vlan`.
+
+|`master`
+|`string`
+|The Ethernet interface to associate with the network attachment. If a `master` is not specified, the interface for the default network route is used.
+
+|`mtu`
+|`integer`
+|Optional: Set the maximum transmission unit (MTU) to the specified value. The default value is automatically set by the kernel.
+
+|`vlanId`
+|`integer`
+|Set the id of the vlan.
+
+|`ipam`
+|`object`
+|The configuration object for the IPAM CNI plugin. The plugin manages IP address assignment for the attachment definition.
+
+Do not specify `dhcp`. Configuring VLAN with DHCP is not supported because VLAN interfaces share the MAC address with the host interface.
+
+|`dns`
+|`integer`
+|Optional: DNS information to return, for example, a priority-ordered list of DNS nameservers.
+
+|`linkInContainer`
+|`boolean`
+|Optional: Specifies if the master interface is in the container network namespace or the main network namespace.
+
+|====
+
+[id="nw-multus-vlan-config-example_{context}"]
+== vlan configuration example
+
+The following example configures an additional network named `vlan-net`:
+
+[source,json]
+----
+{
+ "cniVersion": "0.3.1",
+ "name": "vlan-net",
+ "type": "vlan",
+ "master": "eth0",
+ "mtu": 1500,
+ "vlanId": 5,
+ "linkInContainer": false,
+ "ipam": {
+   "type": "host-local",
+   "subnet": "10.1.1.0/24"
+   },
+  "dns": {
```

Comment:
it seems to me that dns has an extra space. The indentation doesn't seem to be correct

Reply:
resolved